src/measurement/dto/measurement-registration-request.dto.ts

```diff
@@ -13,6 +13,7 @@ export class MeasurementRegistrationRequest
     timezone = ""
     type = ""
     user_server_selection = false
+    capabilities = {"RMBThttp": true}
```

Comment:
I think it would be better to extract it from the `settingsRequest` (line 26 and further), to have a single source of truth.